provisioner/group_vars/all.yml

```diff
@@ -10,6 +10,7 @@ valid_workshop_types:
   - networking
   - f5
   - rhel
+  - security_automation
```

Comment:
Also, please create an example file in `workshops/provisioner/sample_workshops` to highlight the different variables and note if anything special needs to be taken into consideration.